SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -251,23 +306,25 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (DisplayName = "Auto-generate launch configuration file"))
 	bool bGenerateDefaultLaunchConfig;
 
-	/** Returns the Runtime version to use for cloud deployments, either the pinned one, or the user-specified one depending of the settings. */
-	const FString& GetSpatialOSRuntimeVersionForCloud() const;
+	/** Returns which runtime variant we should use. */
+	TEnumAsByte<ESpatialOSRuntimeVariant::Type> GetSpatialOSRuntimeVariant() const { return RuntimeVariant; }
 
-	/** Returns the Runtime version to use for local deployments, either the pinned one, or the user-specified one depending of the settings. */
-	const FString& GetSpatialOSRuntimeVersionForLocal() const;
+	/** Returns the version information for the currently set variant*/
+	const FRuntimeVariantVersion& GetSelectedRuntimeVariantVersion() const
+	{
+		return const_cast<USpatialGDKEditorSettings*>(this)->GetRuntimeVariantVersion(RuntimeVariant);
```

Comment:
I think that function can be const and then we don't need the const cast?

Reply:
Nah the `GetRuntimeVariantVersion` is a helper to get the correct version based on the variant and is used in the Set function which is used by UI